.eslintrc.js

```diff
@@ -10,6 +10,7 @@ module.exports = {
   },
   rules: {
     "comma-dangle": ["error", "never"],
-    complexity: 0
+    complexity: 0,
+    "react/jsx-handler-names": 0
```

Comment:
👍

Reply:
okay, fully agree, dumb way of fighting the system

Reply:
I know we have a lint warning because of this. You can try to fix that if you want (I tried in past but had issues with this one) but its fine to leave it as is since its out of scope.

Reply:
You could also disable it just for the file that causes the warning